include/zephyr/drivers/can.h

```diff
@@ -1159,6 +1159,44 @@ static inline int z_impl_can_stop(const struct device *dev)
 	return api->stop(dev);
 }
 
+/**
+ * @brief Enable CAN controller termination impedance.
+ *
+ * Toggle separate GPIO to enable termination reistor (120ohm) on CAN bus.
+ * This will put 120ohm impendance between CAN_H and CAN_L wires on CAN-PHY's output.
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ * @retval 0 if successful.
+ * @retval -EIO General input/output error, failed to toggle GPIO device.
+ */
+__syscall int can_termination_enable(const struct device *dev);
```

Comment:
In my opinion, termination is not a property of the can controller, but from the bus. There is no need to enable/disable on a state change of the controller.

I would only add an enable to the can transceiver if we agree to add a gpio controlled termination.

The termination is most probably not even part of the transceiver.

Reply:
Thank you. I could remove it from API of course.

Reply:
I removed it from can.h (e.g. CAN API)

Reply:
> The termination is most probably not even part of the transceiver.

Good point. I see termination as  part of physical transport layer and transceiver a tool to control it. Thus maybe it is not wrong to have termination gpis control within phy.